mtp_api/apps/security/tests/test_views/test_check_views.py

```diff
@@ -0,0 +1,1231 @@
+                    'added_by': {
+                        'first_name': self.added_by_user.first_name,
+                        'last_name': self.added_by_user.last_name,
+                        'username': self.added_by_user.username,
+                    }
+                },
+                {
+                    'active': False,
+                    'reason': 'they have shaved it off',
+                    'added_by': {
+                        'first_name': self.added_by_user.first_name,
+                        'last_name': self.added_by_user.last_name,
+                        'username': self.added_by_user.username,
+                    }
+                },
+                {
+                    'active': True,
+                    'reason': 'they grew it back again',
+                    'added_by': {
+                        'first_name': self.added_by_user.first_name,
+                        'last_name': self.added_by_user.last_name,
+                        'username': self.added_by_user.username,
+                    }
+                }
+            ]
+        }
+        post_response = self.client.post(
+            reverse(
+                'security-check-auto-accept-list'
+            ),
+            data={
+                'prisoner_profile': self.prisoner_profile.id,
+                'debit_card_sender_details': self.debit_card_sender_details.id,
+                'states': [
+                    {
+                        'reason': 'they have amazing hair',
+                        'added_by': self.added_by_user.id
+                    }
+                ]
+            },
+            format='json',
+            HTTP_AUTHORIZATION=self.auth,
+        )
+        self.assertEqual(post_response.status_code, 201)
+        auto_accept_rule = post_response.json()
+        patch_response = self.client.patch(
+            reverse(
+                'security-check-auto-accept-detail',
+                args=[auto_accept_rule['id']]
+            ),
+            data={
+                'states': [
+                    {
+                        'active': False,
+                        'reason': 'they have shaved it off',
+                        'added_by': self.added_by_user.id
+                    }
+                ]
+            },
+            format='json',
+            HTTP_AUTHORIZATION=self.auth,
+        )
+        self.assertEqual(patch_response.status_code, 200)
+        patch_response = self.client.patch(
+            reverse(
+                'security-check-auto-accept-detail',
+                args=[auto_accept_rule['id']]
+            ),
+            data={
+                'states': [
+                    {
+                        'active': True,
+                        'reason': 'they grew it back again',
+                        'added_by': self.added_by_user.id
+                    }
+                ]
+            },
+            format='json',
+            HTTP_AUTHORIZATION=self.auth,
```

Comment:
is it a pain to ask for one of these to be for `self.users['security_fiu_users'][1]`? just so we can prove that multiple users' details propagate

Reply:
Not at all, it's a fair point